upgrade/Upgrade-4.0.8.php

```diff
@@ -64,6 +65,12 @@ function upgrade_module_4_0_8($module)
     if (Db::getInstance()->execute($sql) == false) {
         return false;
     }
+    $tabId = Tab::getIdFromClassName('AdminMollieModule');
+    $tab = new Tab($tabId);
+    $tab->name = MultiLangUtility::createMultiLangField('Mollie');
```

Comment:
Strings like 'Molie' or 'AdminMollieModule' could be added to the constants if used in few places, if the string will change in future You'll need to change it only in one place.